src/donationswap.py

```diff
@@ -552,6 +552,21 @@ def confirm_offer(self, secret):
 				offer.confirm(db)
 				eventlog.confirmed_offer(db, offer)
 
+		replacements = {
+			'{%CHARITY%}': offer.charity.name,
+			'{%CURRENCY%}': offer.country.currency.iso,
+			'{%AMOUNT%}': offer.amount,
+			'{%MIN_AMOUNT%}': offer.min_amount,
+			'{%COUNTRY%}': offer.country.name
+		}
+
+		self._mail.send(
+			util.Template('email-subjects.json').json('post-confirmed-email'),
+			util.Template('post-confirmed-email.txt').replace(replacements).content,
+			html=util.Template('post-confirmed-email.html').replace(replacements).content,
+			to=self._config.contact_message_receivers['to']
```

Comment:
todolist item: Catherine would like an email to donationswap@eahub.org when a new offer is made (low priority)